src/pixee-platform.ts

```diff
@@ -6,29 +6,27 @@ import { TOOL_PATH } from "./inputs";
 import { getGitHubContext, getRepositoryInfo } from "./github";
 
 export async function uploadInputFiles(tool: TOOL_PATH, files: Array<string>) {
-  const form = new FormData();
-
   const path = require("path");
-
-  // Append each file to the form data
-  files.forEach((file) => {
-    form.append("files", fs.readFileSync(file), path.basename(file));
-  });
-
   const pixeeUrl = core.getInput("pixee-api-url");
   const token = await core.getIDToken(pixeeUrl);
   const url = buildUploadApiUrl(tool);
 
-  return axios
```

Comment:
The action module expects to `await` on the promise returned by this method. This change eliminates the returning of the promise.

You'll want to flatten all the promises from each request into one promise and return that.